renku/utils/doi.py

```diff
@@ -34,7 +34,5 @@ def extract_doi(uri):
     """Return the DOI in a string if there is one."""
     match = doi_regexp.match(uri)
 
-    if match is None:
-        return None
-
-    return match.group(2)
+    if match:
+        return match.group(2)
```

Comment:
I prefer explicit return of None instead of implicit, not sure if we have any conventions regarding this.

Reply:
Honestly, we have both messes in the repo. I would never say that this is explicit but just plain verbose. We should pick one style and stick with it for future reviews, and since I'm not in favor of adding `return None` to all unit functions I would vote for non-verbose option.